doc/ref_model/chapters/chapter03.md

```diff
@@ -123,7 +123,7 @@ _**Example**: a tenant within an OpenStack environment or a Kubernetes cluster._
 
 <a name="3.2.1.2"></a>
 #### 3.2.1.2 Virtual Compute
-A virtual machine or a container/pod is used by a tenant capable of hosting the application components of workloads (VNFs). A virtual compute therefore requires a tenant context and, since it will need to communicate with other communication partners, it is assumed that the networks have been provisioned in advance.
+A virtual machine or a container/pod capable of hosting the application components of workloads (VNFs/CNFs) of the Tenant. A virtual compute therefore requires a tenant execution context and, since it will need to communicate with other communication partners, it is assumed that the networks have been provisioned in advance.
```

Comment:
The first sentence lost the verb.  Should it be: A virtual machine or a container/pod is capable of hosting the application components of workloads (VNFs/CNFs) of the Tenant. 
"Tenant"or "tenant"?  Inconsistency between first and second sentence.

Reply:
@walterkozlowski corrected.